charts/resources/instance/templates/instance.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  name: {{ .Values.name }}
+spec:
+  serviceClassName: {{ .Values.serviceClassName }}
+  planName: {{ .Values.planName }}
```

Comment:
Could we add parameters here to see how they would line up? Right now this seems like a lot of overhead to create a chart and a helm install, when a simple kubectl create would suffice.